### PR TITLE
Deprecate metrics and settings related to Akka

### PR DESCRIPTION
**Pull request by akarasavov**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1731.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/configuration/configuration-settings)
- [Metrics reference](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1731.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/configuration/configuration-settings)
- [Metrics reference](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/monitoring/metrics/reference)

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 1]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
 [[discovery-service-V1]]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:425]:
```suggestion
[role=label--deprecated-5.23]
[[discovery-service-V1]]
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 2]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
...
 === Discovery metrics v1
 
+<<<<<<< Updated upstream
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:428]:
```suggestion
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 3]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
...
 .Discovery service V1
+=======
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:430]:
```suggestion
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 4]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
...
+label:deprecated[Deprecated in 5.23]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:431]:
```suggestion
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 5]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
...
+
+.Discovery metrics
+>>>>>>> Stashed changes
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:434]:
```suggestion
```

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 6]
```diff
@@ -425,7 +425,13 @@ label:deprecated[Deprecated in 5.15]
 [[discovery-service-V1]]
 === Discovery metrics v1
 
+<<<<<<< Updated upstream
 .Discovery service V1
+=======
+label:deprecated[Deprecated in 5.23]
+
+.Discovery metrics
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:433]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1731.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/configuration/configuration-settings)
- [Metrics reference](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1731.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/configuration/configuration-settings)
- [Metrics reference](https://neo4j-docs-operations-1731.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Review by NataliaIvakina — APPROVED** [review]
Everything looks good from my side. If those changes are in the code already, we can safely merge this PR.

**Pull request merged**